src/pivot-table/hooks/use-scroll.ts

```diff
@@ -77,7 +125,7 @@ const useScroll = ({ layoutService, pageInfo, mockedRefs }: Props) => {
     if (!(evt.target instanceof HTMLDivElement)) return;
 
     if (leftGridRef.current) {
-      leftGridRef.current.forEach((list) => list.scrollTo(evt.currentTarget.scrollTop));
+      leftGridRef.current.filter(Boolean).forEach((list) => list.scrollTo(evt.currentTarget.scrollTop));
```

Comment:
Why the `filter` here?

Reply:
because when we expand a node we get this list also expanded (from `list: [{col#01}]` to `list: [{col#01}, {col#02}, ...]`)

when we collapse the nodes -> the list is not shrinking down to `list: [{col#01}]`, but it stays like `list: [{col#01}, null, ...]`). So that null values causes this to throw error complaining about `null.scrollTo(...)` does not exists and this filter boolean just skips those null values

Reply:
Has that changed? Has the`topGridRef` not the same issue?

Reply:
top grid is working fine, but that only was in case of leftGrid

https://github.com/qlik-oss/sn-pivot-table/assets/29652890/80e3d7f3-93b5-4ec1-a9ef-1aad0a7c8ec2